Formula/skaa.rb

```diff
@@ -0,0 +1,35 @@
+class Skaa < Formula
```

Comment:
I wonder if we should call this formula `seven-kingdoms` instead.
```suggestion
class SevenKingdoms < Formula
```
We can then add an alias called `7kaa` so that users can either `brew install seven-kingdoms` or `brew install 7kaa`.

Reply:
@carlocab That sounds nice. It's the first time for me making a formula, so I didn't know about aliases.